packages/manager/apps/hpc-vmware-managed-vcd/src/utils/veeamBackupId.ts

```diff
@@ -0,0 +1,9 @@
+import IVcdOrganization from '@/types/vcd-organization.interface';
+
+const getOrganizationUuid = (organization?: IVcdOrganization) =>
+  organization?.id.split(
+    `${organization?.currentState.region.toLowerCase()}-`,
+  )[1];
```

Comment:
I do the bad man. But we have TU for this utils functions ?

We don't try anticipate if id is bad formated ? (I don't known why)

Reply:
It will be in the shared module with veeam/vcd and is tested there